cranelift/codegen/src/isa/x64/abi.rs

```diff
@@ -89,7 +89,7 @@ impl ABIMachineSpec for X64ABIMachineSpec {
         mut args: ArgsAccumulator<'_>,
     ) -> CodegenResult<(u32, Option<usize>)>
     where
-        I: IntoIterator<Item = &'a ir::AbiParam>,
+        I: IntoIterator<Item = &'a ir::AbiParam> + Clone,
```

Comment:
Is this still necessary? I'm not seeing any `.clone()`s here.

Reply:
Oops, indeed!